rotkehlchen/chain/ethereum/modules/nfts.py

```diff
@@ -132,35 +156,43 @@ def get_all_info(
             entries_limit=FREE_NFT_LIMIT,
         )
 
+    def get_single_nft(self, nft_id: str) -> Optional[Dict[str, Any]]:
+        with self.db.conn.read_ctx() as cursor:
+            cursor.execute(NFT_INFO_SQL_QUERY + ' WHERE identifier = ?', (nft_id,))
```

Comment:
Why not use the `nft_id` filter you added? 
If you're not using it here, then there's no need to have it.

Correct me if I'm wrong but you can use `filter_query.prepare(with_pagination=False)`. With changing `lps_handling` to optional, it should be easy to use.